### PR TITLE
[8.1] PilotAgentsDB: move DN in favor of username

### PR DESCRIPTION
**Pull request by fstagni**
- [x] update wiki

This PR simplifies the PilotsAgentsDB content, eliminating information that is most probably not useful.

BEGINRELEASENOTES

*WMS
CHANGE: PilotAgentsDB: move DN in favor of username

ENDRELEASENOTES

### CODE REVIEW

**Comment by martynia** [comment]
I'm wondering if `OwnerGroup` is really obsolete. I'm using it to get a `VO` of a pilot (this is basically an extension to `export_getPilotOutput()`:

```
       # get pilot stamp by pilot reference:
        result = self.pilotAgentsDB.getPilotInfo(pilotReference)
        if not result["OK"]:
            self.log.error("Failed to get info for pilot", result["Message"])
            return S_ERROR("Failed to get info for pilot")
        if not result["Value"]:
            self.log.warn("The pilot info is empty", pilotReference)
            return S_ERROR("Pilot info is empty")

        pilotDict = result["Value"][pilotReference]
        pilotStamp = pilotDict["PilotStamp"]
        owner = pilotDict["OwnerDN"]
        group = pilotDict["OwnerGroup"]
        vo = getVOForGroup(group)

```

I need the VO to get to the correct storage area, when retrieving the logs (#7105)

**Comment by fstagni** [comment]
Well, indeed we need a way to recognize for which VO pilots have been sent. `OwnerGroup` can be an option. I will restore it.

**Comment by fstagni** [comment]
(final) review of this one before tomorrow's hacakathon?

File: release.notes [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 *TransformationSystem
 
-NEW: (#7170) 
+NEW: (#7170)
```
Comment (andresailer) [inline release.notes]: Rather the line-break is wrong.
```suggestion
NEW: (#7170) Implement a finer grained permissions model for Transformations.
```

File: src/DIRAC/Interfaces/scripts/dirac_admin_get_job_pilots.py [hunk 2]
```diff
@@ -59,7 +48,7 @@ def main():
             exitCode = 2
 
     for error in errorList:
```
Comment (andresailer) [inline src/DIRAC/Interfaces/scripts/dirac_admin_get_job_pilots.py]:
```suggestion
    for job, error in errorList:
```

File: src/DIRAC/Interfaces/scripts/dirac_admin_get_job_pilots.py [hunk 3]
```diff
@@ -59,7 +48,7 @@ def main():
...
-        print("ERROR %s: %s" % error)
+        print(f"ERROR {error}")
```
Comment (andresailer) [inline src/DIRAC/Interfaces/scripts/dirac_admin_get_job_pilots.py]:
```suggestion
        print(f"ERROR {job}: {error}")
```

File: src/DIRAC/Interfaces/scripts/dirac_admin_get_pilot_info.py [hunk 4]
```diff
@@ -100,7 +89,7 @@ def main():
                 print(diracAdmin.pPrint.pformat({gridID: res}))
 
     for error in errorList:
-        print("ERROR %s: %s" % error)
+        print(f"ERROR: {error}")
```
Comment (andresailer) [inline src/DIRAC/Interfaces/scripts/dirac_admin_get_pilot_info.py]: as above. `error` is a tuple

**Review by aldbr — APPROVED** [review]
Looks good to me!

**Pull request merged**